extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -387,9 +374,25 @@ export class Bind {
       }
       const element = dev().assertElement(node);
       const tagName = element.tagName;
-      if (DYNAMIC_TAGS[tagName]) {
-        this.observeElementForMutations_(element);
+
+      let dynamicElements = [];
+      if (typeof element.getDynamicElements === 'function') {
+        dynamicElements = element.getDynamicElements();
+      } else if (element.tagName === 'FORM') {
+        // FORM is not an amp element, so it doesn't have the getter.
```

Comment:
We could add a static method to AmpForm that returns the related instance for a given form element. Similar to what we do with [`Resource#forElement`](https://github.com/ampproject/amphtml/blob/b341d269ce4b032918137e82312ac9c8cad8cc4b/src/service/resource.js#L81-L93)